### PR TITLE
feat: Added the WebXTeam page

### PR DESCRIPTION
**Pull request by Vaani-pathariya**
## Related Issue
This pull request is to add the page : WebX team page

Closes: #321 

### Describe the changes you've made
Added two pages , one to display the team page and the other is a person card .

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [ ] This change requires a documentation update

## How Has This Been Tested?
Have tested it on my local development environment

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

|      Original       |      Updated       |
| :-----------------: | :----------------: |
| Initially their was a 404 page | image
## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/WebXDAO/.github/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by vinzvinci — COMMENTED** [review]
Thanks for contributions! Tagged other mentors and developers guild to review this PR. Expect the review this coming weekdays!

**Review by Anmol-Baranwal — CHANGES_REQUESTED** [review]
@vinzvinci,
Could you please provide your thoughts on these recommended changes?

I recommend making the following changes:

1. Increase the gap significantly between the footer and the last element to create better visual separation.

image

2. Address the responsiveness of the website to ensure it adapts well to different screen sizes.

image

3. Hide the horizontal overflow with scrollbar to prevent content from extending beyond the visible area.

image

**Comment by mkubdev** [comment]
Hello @Vaani-pathariya :wave:

Thank you for your contribution 🎉 Do you need help to add changes from our mentors @Anmol-Baranwal ?
If so, don't hesite to tell us!

**Comment by Vaani-pathariya** [comment]
> > Hello @Vaani-pathariya 👋
> > Thank you for your contribution 🎉 Do you need help to add changes from our mentors @Anmol-Baranwal ? If so, don't hesite to tell us!
> 
> @Vaani-pathariya I am glad to help with anything. Please tag me here, and I will assist you with whatever you need.

Thankyou for the help, I am working on the requested changes , will open pr soon

**Comment by Vaani-pathariya** [comment]
I have done the changes , please check

File: app/teams/page.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      <TeamCard Name="Shubham Kukreti  " position="WebX Guild Core" github_url="" linkedin_url="" twitter_url=""/>
+      <TeamCard Name="Ritvik Shukla" position="WebX Guild Core" github_url="" linkedin_url="" twitter_url=""/>
+      <TeamCard Name="Maxime Kubik" position="WebX Guild Core" github_url="" linkedin_url="" twitter_url=""/>
+      </div>
+      <h1 className="text-center mt-10 font-bold">WebX Guild Maintainers and Moderators</h1>
+      <h1 className="text-center mt-5">The essential role of our community projects &apos; maintainers and moderators is to ensure the <br/>proper functioning of our codebase and foster a welcoming and engaged community.</h1>
+      <div className="container mx-auto justify-center gap-10 mt-10 mb-10 lg:flex sm:block">
+      <TeamCard Name="David Leal" position="WebX Guild Core" github_url="" linkedin_url="" twitter_url=""/>
+      <TeamCard Name="Krish Gupta" position="WebX Guild Core" github_url="" linkedin_url="" twitter_url=""/>
+      <TeamCard Name="Abhinav MV" position="WebX Guild Core" github_url="" linkedin_url="" twitter_url=""/>
```
Comment (Panquesito7) [inline app/teams/page.tsx:50]: I think the positions (instead of using `WebX Guild Core`) here should be changed.

**Review by Anmol-Baranwal — COMMENTED** [review]
@Panquesito7 @vinzvinci 
Please mark the points that you believe should be implemented. Once those points are approved, please proceed with the tasks they mention.

<hr>

There can be improvements-

1. There is excessive negative space. Instead of having only one column at `1023px`, we could consider utilizing two columns when it seems appropriate.

image

2. The line spacing is the same for both icons and text, which disrupts the visual hierarchy.

image

3. Please provide some margin space on the left side.

image

<hr>

The overall structure and responsiveness are satisfactory, and these changes would be minor adjustments.

> You can use grid: `e.g. md:grid md:grid-cols-2` for appropriate structure

If you are stuck, do let me know.

**Comment by vinzvinci** [comment]
Points already marked on this one

**Comment by Vaani-pathariya** [comment]
Did the required changes , please check

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Review by vinzvinci — APPROVED** [review]
LGTM!

**Pull request merged**